pkg/gcp/build/build.go

```diff
@@ -104,14 +98,28 @@ func (o *Options) uploadBuildDir(targetBucket string) (string, error) {
 	defer os.Remove(name)
 
 	logrus.Infof("Creating source tarball at %s...", name)
-	if err := runCmd("tar", "--exclude", ".git", "-czf", name, "."); err != nil {
+	tarCmdErr := command.Execute(
+		"tar",
+		"--exclude",
+		".git",
+		"-czf",
+		name,
+		".",
+	)
+	if tarCmdErr != nil {
```

Comment:
Nitpick: do you prefer not to check these errors inline? It doesn’t appear that the error value is reused after check for not nil here

Reply:
@hasheddan -- I had a slight preference towards this since it makes the command a little more readable not being in the `if` clause line.